hello_world/blockchain_app/hello_plugin/index.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const cors = require("cors");
+const { BasePlugin } = require("lisk-sdk");
+const pJSON = require("../package.json");
+
+class HelloAPIPlugin extends BasePlugin {
+  _server = undefined;
+  _app = undefined;
+  _hello = undefined;
+
+  static get alias() {
+    return "HelloHTTPAPI";
+  }
+
+  static get info() {
+    return {
+      author: pJSON.author,
+      version: pJSON.version,
+      name: pJSON.name,
+    };
+  }
+
+  get defaults() {
+    return {};
+  }
+
+  get events() {
+    return [];
+  }
+
+  get actions() {
+    return {};
+  }
+
+  async load(channel) {
+    this._app = express();
+
+    channel.subscribe('hello:newHello', (info) => {
+      this._hello = info;
+    });
+
+    channel.once("app:ready", () => {
+      this._app.use(cors({ origin: "*", methods: ["GET", "POST", "PUT"] }));
+      this._app.use(express.json());
+
+      this._app.get("/api/hello_counter", async (_req, res) => {
+        const counter = await channel.invoke("hello:amountOfHellos");
+
+        await res.json({ data: counter });
+      });
+
+      this._app.get("/api/latest_hello", async (req, res) => {
+        await res.json(this._hello);
+      });
```

Comment:
This will not return latest hello for the case application is restarted and no new transaction is sent to node.

Reply:
Hmm ok... How can it be changed to be persistent?

Reply:
There can be two ways: 

1. Create an action in the module which return latest hello string by looking at the latest helloAsset transaction 
2. Make assumption and document that his endpoint only return latest hello in current session since the node was not restarted.

Reply:
I added a comment in the hello plugin